### PR TITLE
techdebt cleanup

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Just come cleanup i noticed while working on other ticket

### CODE REVIEW

File: src/Components/BureauPage/BureauPage.jsx [hunk 1]
```diff
@@ -34,9 +24,7 @@ const BureauPage = () => {
         <Route path="/profile/ao/panelmeetings" render={() => <PanelMeetingSearch isCDO={false} />} />
         <Route path="/profile/ao/availablebidders" render={() => <AvailableBidderContainer isCDO={false} isAO />} />
         <Route path="/profile/ao/panelmeetingagendas/:pmID" render={() => <PanelMeetingAgendas isAO />} />
-        <Route path="/profile/(bureau|ao)/dashboard" render={() => <Dashboard {...dashboardProps} />} />
```
Comment (elizabeth-jimenez) [inline src/Components/BureauPage/BureauPage.jsx:37]: changes i missed during the cleanup of Static Content

https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2558/files
you can dbl check the menu items i removed in that PR with the routes i removed in this one

File: src/Components/PostPage/PostPage.jsx [hunk 2]
```diff
@@ -18,7 +13,6 @@ const PostPage = props => {
   return (
     <div className="usa-grid-full profile-content-container">
       <Switch>
-        <Route path="/profile/post/dashboard" render={() => <Dashboard {...dashboardProps} />} />
```
Comment (elizabeth-jimenez) [inline src/Components/PostPage/PostPage.jsx:21]: [same as](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2594/files#r1240334377)

File: src/Components/ProfileDashboard/UserProfile/UserProfileGeneralInformation/UserProfileGeneralInformation.jsx [hunk 3]
```diff
@@ -81,6 +81,7 @@ class UserProfileGeneralInformation extends Component {
           </div>
           <div className="name-group">
             <SectionTitle small title={`${userProfile.user.last_name ? `${userProfile.user.last_name}, ` : ''}${userProfile.user.first_name}`} className="current-user-name" />
+            :)
```
Comment (elizabeth-jimenez) [inline src/Components/ProfileDashboard/UserProfile/UserProfileGeneralInformation/UserProfileGeneralInformation.jsx:84]: :)

File: src/Constants/Menu.js [hunk 4]
```diff
@@ -42,7 +42,6 @@ function MenuConfig(config) {
 export const GET_PROFILE_MENU = () => MenuConfig([
   {
     text: 'Profile',
-    route: '/profile/dashboard/',
```
Comment (elizabeth-jimenez) [inline src/Constants/Menu.js:45]: I had such an epiphany with these! 🧠 
I never really looked closely at menu items, really just would add them and move on, but i'm pretty sure this route is not needed bc toggleMenuSection takes precedent and instead just toggles the children, so clicking this is never taking us to that route, it's just expanding/collapsing sub-menu

Comment (elizabeth-jimenez) [reply src/Constants/Menu.js:45]: but there is a comment in NavLink that i want to follow up on 🤔

Comment (elizabeth-jimenez) [reply src/Constants/Menu.js:45]: followed up on - comment was only relevant to hidden prop

File: src/Components/ProfileDashboard/UserProfile/UserProfileGeneralInformation/UserProfileGeneralInformation.jsx (hunk 3 above)
Comment (pveland621) [reply src/Components/ProfileDashboard/UserProfile/UserProfileGeneralInformation/UserProfileGeneralInformation.jsx]: :)

**Review by scott062 — APPROVED** [review]

**Pull request merged**